Menu/Tab/index.jsx

```diff
@@ -82,6 +84,16 @@ export default React.createClass({
     }
   },
 
+  componentWillUnmount () {
+    window.removeEventListener('resize', this.resizeListener)
+  },
+
+  resizeListener () {
+    window.requestAnimationFrame
+      ? window.requestAnimationFrame(() => update(this))
```

Comment:
Done. My quick test showed it working. Whether or not this is making a difference I am not sure how to test (that is, without investing a lot of time ;) )